docs/features/internal_libraries.md

```diff
@@ -0,0 +1,50 @@
+# Internal Libraries
+
+```js
+import fek // import imports an internal library as a special record of the same name into the program
+// fek.paer("hi") depricated, as the fek library was a test that was left in by accident
+
+import FileIO // Used for File input and output
+FileIO.write("test.txt", "test")! // This overwrites everything inside the test.txt file, it also creates a new file if it does not exist
```

Comment:
Since we're using `!`, might want to clarify somewhere that these are in a function that returns a `cmd`?